lib/response.js

```diff
@@ -178,6 +178,16 @@ Response.prototype.getHeaders = function getHeaders() {
 };
 Response.prototype.headers = Response.prototype.getHeaders;
 
+/**
+ * Headers that cannot be multi-values.
+ * @see #779, don't use comma separated values for set-cookie
+ * @see #986, don't use comma separated values for content-type
+ * @see http://tools.ietf.org/html/rfc6265#section-3
+ */
+var HeaderArrayBlacklist = {
```

Comment:
Since this is a constant, can you cap case it and move it to the top of the file? Under the `/// Globals` section.